### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by SarifIslam**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,5 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Welcome to Potionfy</title>
   </head>
-  <body></body>
+  <body>
+    <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (SalahuddinAhammed) [inline index.html:10]: Can you remove the string `Enjoy your stay` and make the string `Welcome to Potionfy` red?

Comment (SarifIslam) [reply index.html:10]: Sure, can do.

**Review by SalahuddinAhammed — CHANGES_REQUESTED** [review]
Please change the code as per my comment in the review.

**Review by SalahuddinAhammed — APPROVED** [review]
Well done, @SarifIslam.

**Pull request merged**